srcpkgs/python3-iced-x86/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-iced-x86'
+pkgname=python3-iced-x86
+version=1.21.0
+revision=1
+build_style=python3-module
+build_helper=rust
+hostmakedepends="python3-setuptools-rust cargo"
+depends="python3"
+short_desc="Fast and correct x86/x64 disassembler, assembler, decoder, encoder"
+maintainer="Mihail Ivanchev <contact@ivanchev.net>"
+license="MIT"
+homepage="https://github.com/icedland/iced"
+distfiles="https://github.com/icedland/iced/archive/refs/tags/v${version}.tar.gz"
+checksum=abd6aab6393283bd435051ccb4b00d432bb4fd548832a7905dd87d815b72685c
+build_wrksrc="src/rust/iced-x86-py"
+
+if [ "$CROSS_BUILD" ]; then
+	makedepends+=" rust-std"
```

Comment:
Just put them into makedepends without the condition, this will be installed through rust on the host anyways.